pdf/core/crypt_file_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+// Integration tests for the PDF crypt support.
+
+package core_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	pdfcontent "github.com/unidoc/unidoc/pdf/contentstream"
+	pdf "github.com/unidoc/unidoc/pdf/model"
+)
+
+const aes3Dir = `../../testfiles/AESv3`
```

Comment:
Place in testdata under package folder https://github.com/unidoc/unidoc/wiki/UniDoc-Developer-Guide#14-test-and-supplementary-files